episodes/superspreading-estimate.Rmd

```diff
@@ -0,0 +1,674 @@
+
+the estimated value of k
+suggest that
+SSE are an expected feature of the
+individual level variation in infectiousness
+
+
+
+```{r}
+library(fitdistrplus)
+
+## fit distribution
+offspring_fit <- epi_secondary %>%
+  pull(secondary_cases) %>%
+  fitdist(distr = "nbinom")
+
+offspring_fit
+
+## extract the "size" parameter
+mid <- offspring_fit$estimate[["size"]]
+```
+
+```{r,echo=FALSE}
+## calculate the 95% confidence intervals using the standard error estimate and
+## the 0.025 and 0.975 quantiles of the normal distribution.
+lower <- mid + offspring_fit$sd[["size"]] * qnorm(0.025)
+upper <- mid + offspring_fit$sd[["size"]] * qnorm(0.975)
+```
+
+We can see that the
+dispersion parameter is estimated as
+`r round(mid, 3)` (95% CI `r `round(lower, 3)` - `r round(upper, 3)`).
+
+As this value is significantly lower
+than one, we can conclude that
+the degree of super-spreading is high.
+
+This is in line with visual inspection
+of the histogram made above.
+
+```{r,echo=FALSE}
+# calculate density fit
+fit_density <-
+  tibble(quantile = 0:40) %>%
+  mutate(
+    density = dnbinom(
+      x = quantile,
+      mu = offspring_fit$estimate[["mu"]],
+      size = mid
+    )
+  )
+
+# plot offspring distribution with density fit
+ggplot() +
+  geom_histogram(
+    data = epi_secondary,
+    mapping =
+      aes(
+        x = secondary_cases,
+        y = after_stat(density)
+      ),
+    binwidth = 1
+  ) +
+  geom_point(
+    data = fit_density,
+    mapping = aes(x = quantile, y = density),
+    alpha = 0.3, color = "red"
+  ) +
+  labs(
+    xlab = "Individual reproduction number",
+    ylab = "Number of individuals"
+  )
+```
+
+::::::::::::::::::::::: callout
+
+### Transmission heterogeneity
+
+Heterogeneity is in the variance of the number of secondary cases generated by each infected individual
+
+[add table quick summary]
+
+:::::::::::::::::::::::
+
+::::::::::::::::::::::: challenge
+
+read ebola contact data and estimate R and dispersion?
+
+:::::::::::::::::::::::
+
+## Interpret the dispersion parameter
+
+<!-- from draft -->
+
+- for occurrences of associated discrete events we can use Poisson or negative binomial discrete distributions.
+- in a Poisson distribution, mean is equal to variance.
+- when variance is higher than the mean, this is called over-dispersion.
+- in biological applications, over-dispersion occurs and so a negative binomial may be worth considering as an alternative to Poisson distribution.
+
+- in epidemiology, negative binomial have being used to model disease transmission for infectious diseases where the likely number of onward infections may vary considerably from individual to individual and from setting to setting
+
+- In a sequence of independent Bernoulli trials, in each trial the probability of success is $p$ and of failure is $1-p$. We observe this sequence until a predefined number $r$ of successes occurs. Then the random number of observed failures follows the negative binomial distribution.
```

Comment:
I adapted some of this content to this spoiler with the aim of connecting the concepts of poisson, overdispersion and NB for the ones that needed it. You assessment of this will also be helpful

https://github.com/epiverse-trace/tutorials-middle/blob/3e536a4a6406e5618c8e102c984ff6a9dbfe40f9/episodes/superspreading-estimate.Rmd#L244-L258